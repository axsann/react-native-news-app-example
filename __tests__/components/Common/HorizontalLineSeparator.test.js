// @flow

import 'react-native'
import React from 'react'
import HorizontalLineSeparator from '../../../src/components/Common/HorizontalLineSeparator.component'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

describe('HorizontalLineSeparator', () => {
  it('余白あり', () => {
    const instance = renderer.create(
      <HorizontalLineSeparator />
    )

    expect(instance.toJSON()).toMatchSnapshot()
  })

  it('余白なし', () => {
    const instance = renderer.create(
      <HorizontalLineSeparator noMargin={true} />
    )

    expect(instance.toJSON()).toMatchSnapshot()
  })
})
