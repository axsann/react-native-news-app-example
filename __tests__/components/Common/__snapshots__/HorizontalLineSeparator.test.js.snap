// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HorizontalLineSeparator 余白あり 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#eeeeee",
        "height": 1,
      },
      Object {
        "marginLeft": 10,
        "marginRight": 10,
        "marginTop": 10,
      },
    ]
  }
/>
`;

exports[`HorizontalLineSeparator 余白なし 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#eeeeee",
        "height": 1,
      },
      Object {
        "marginLeft": 0,
        "marginRight": 0,
        "marginTop": 0,
      },
    ]
  }
/>
`;
